src/game.cpp

```diff
@@ -5279,6 +5277,41 @@ void game::draw_sidebar()
     draw_sidebar_messages();
 }
 
+void game::draw_safe_mode()
```

Comment:
Passing in the window to allow for decoupling there makes sense. As for passing in the x/y, would that not require pulling out some of the internal logic happening in the `draw_safe_mode()` function? The y / line perhaps, but would the x represent the left-aligned start or right-aligned start? If left aligned, I would think then the calling code would already have to know about the translated SAFE string and length, no?